### ISSUE TITLE
Where can I find Documentation or Examples?

### ISSUE THREAD

**Issue by jobenscott** [leejarvis/gridhook#13]
SendGrid's "Event Webhook" is the only way to see stats from existing emails, from what I understand. I first looked into SendGrid's supported gem eventkit-rails. Unfortunately, THERE WAS NO DOCUMENTATION OR EXAMPLES. So, I saw this gridhook gem was one of the dependencies. Upon reading your READ.ME, it seemed this gem was sufficient. Unfortunately, AGAIN, this gem has no documentation or examples. There was an issue added MORE THAN 2 YEARS AGO indicating your Documentation link is broken. No response. I see this was last committed to 28 days ago, so I assume it's still active. I'd really appreciate if you could either: direct me to some type of documentation or example, or direct me to an alternative I can use. Thanks.

**Comment by skatkov** [comment]
Examples are here:
https://github.com/leejarvis/gridhook/blob/master/README.md

**Comment by jobenscott** [comment]
@skatkov 
Yes, I've reviewed this. I've added the configuration to /initializers/sendgrid.rb.

I added a new model sendgrid.rb. Inside that file I've added the 
`Class EventProcessor < ApplicationRecord
    def call(event)
         puts event.to_yaml
    end
end`

I'm not sure how I am supposed to call the `call()` function. Am I misunderstanding something, or is there more documentation on how to achieve this? Also, I haven't passed any credentials to gridhook. Do I need to do this?
